README.md

```diff
@@ -58,10 +58,10 @@ and are easy to use. Either through a
 
 The body of the request should contain the following parameters:
 
-| name | mandatory | description | default |
-|------|-----------|-------------|---------| 
-| apimApiName | yes | The name of the API in Azure API Manager | n/a |
-| searchApiVersion | no | The version identifier for the Azure Search API | 2017-11-11 |
+| Name | Required? | Description | Default | Allowable Values |
```

Comment:
Nit: The question mark at the end of `Required` isn't required?
Nit: In the tables on other READMEs, `Required` has been later in the table. I think it would probably fit better after `Description` in this table or even as the last column.